docs/data/material/getting-started/usage/usage.md

```diff
@@ -1,39 +1,40 @@
 # Usage
 
-<p class="description">Get started with React and Material UI in no time.</p>
+<p class="description">Learn the basics of working with Material UI components.</p>
 
-Material UI components work in isolation.
-**They are self-supporting**, and will only inject the styles they need to display.
-They don't rely on any global style-sheets such as [normalize.css](https://github.com/necolas/normalize.css/).
+## Quickstart
 
-You can use any of the components as demonstrated in the documentation.
-Please refer to each component's [demo page](/material-ui/react-button/) to see how they should be imported.
-
-## Quick start
-
-Here's a quick example to get you started, **it's literally all you need**:
+The following code snippet demonstrates a simple app that uses the Material UI [Button](/material-ui/react-button/) component:
 
 ```jsx
 import * as React from 'react';
 import Button from '@mui/material/Button';
 
-function App() {
-  return <Button variant="contained">Hello World</Button>;
+export default function MyApp() {
+  return (
+    <div>
+      <Button variant="contained">Hello World</Button>
+    </div>
+  );
 }
 ```
 
-Yes, this really is all you need to get started, as you can see in this live and interactive demo:
+You can play around with this code in the interactive Code Sandbox demo below.
+Try changing the `variant` on the Button to `outlined` to see how the style changes:
 
 {{"demo": "Usage.js", "hideToolbar": true, "bg": true}}
 
 ## Globals
 
-Material UI usage experience can be improved with a handful of important globals that you'll need to be aware of.
+Since Material UI components are built to function in isolation, they don't require any kind of globally scoped styles.
+
+For a better user experience and developer experience, we recommend adding the following globals to your app.
```

Comment:
This would personally be easier for me to read:

```suggestion
Since Material UI components are built to function in isolation, they don't require any kind of globally scoped styles.
But for a better user and developer experience, we recommend adding the following globals to your app.
```